cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/MapChangeParsingStrategy.java

```diff
@@ -41,18 +50,28 @@
         return Arrays.asList(MapChange.class);
     }
 
-    private CfMapValueChanged parseMapEntry(EntryChange entryChange) {
+    private CfMapValueChanged parseMapEntry(String propertyName, EntryChange entryChange) {
         if (entryChange instanceof EntryAdded) {
+            log.verbose("Appending", propertyName,"map change with added entry:"
+                    , entryChange.getKey() + ":", ((EntryAdded) entryChange).getValue());
+
             return new CfMapValueChanged(entryChange.getKey().toString(),
                     "",
                     ((EntryAdded) entryChange).getValue().toString(),
                     ChangeType.ADDED);
         } else if ( entryChange instanceof EntryRemoved) {
+            log.verbose("Appending", propertyName,"map change with removed entry:",
+                    entryChange.getKey() + ":", ((EntryRemoved) entryChange).getValue());
```

Comment:
I would recommend to use `debug` logging here to prevent long and confusing output